client/coral-framework/graphql/mutations/index.js

```diff
@@ -171,3 +172,22 @@ export const stopIgnoringUser = graphql(STOP_IGNORING_USER, {
     };
   }
 });
+
+export const editComment = graphql(EDIT_COMMENT, {
+  props: ({mutate}) => {
+    return {
+      editComment: (id, asset_id, edit) => {
+        return mutate({
+          variables: {
+            id,
+            asset_id,
+            edit,
+          },
+          refetchQueries: [
+            'EmbedQuery'
```

Comment:
@cvle If you edit a comment that's APPROVED, and that sets the state back to something like PREMOD or REJECTED, then the result of EmbedQuery will no longer include the edited comment.

> We don't want to rely on refetching for mutations.

Why not? What's a better tool for the job?